### ISSUE TITLE
Is there a way to retrieve a video from adultswim that requires a cable login?

### ISSUE THREAD

**Issue by jluaol** [streamlink/streamlink#1656]
----

### Checklist

- [ ] This is a bug report.
- [x] This is a feature request.
- [ ] This is a plugin (improvement) request.
- [ ] I have read the contribution guidelines.

### Description

I have a cable login which works on the website, but would like to use streamlink to get the video. I've downloaded all of the free episodes so far, with no problems. Is there any way this can be accomplished?

### Explicit stream URLs to test

1. http://www.adultswim.com/videos/rick-and-morty/mortys-mind-blowers/
2. http://www.adultswim.com/videos/rick-and-morty/the-ab-cs-of-beth/
3. http://www.adultswim.com/videos/rick-and-morty/the-rickchurian-mortydate/

### Logs

```
[cli][debug] OS:         Windows 10
[cli][debug] Python:     3.6.5
[cli][debug] Streamlink: 0.12.1
[cli][debug] Requests(2.18.4), Socks(1.6.7), Websocket(0.47.0)
[cli][info] Found matching plugin adultswim for URL http://www.adultswim.com/videos/rick-and-morty/mortys-mind-blowers/
[plugin.adultswim][debug] Loading VOD streams for: rick-and-morty/mortys-mind-blowers
error: Unable to open URL: /secure/tve/2018-04/ASOE1003161800083222_HLS/ASOE1003161800083222_iphone_vod.m3u8 (Invalid URL '/secure/tve/2018-04/ASOE1003161800083222_HLS/ASOE1003161800083222_iphone_vod.m3u8': No schema supplied. Perhaps you meant http:///secure/tve/2018-04/ASOE1003161800083222_HLS/ASOE1003161800083222_iphone_vod.m3u8?)
```

**Comment by beardypig** [comment]
Looks like there is another bug going on there too. But, with regards to cable logins; we have discussed it before and we decided before that it would not be feasible to support. As far as I know each cable provider has their own login page, and it uses some kind of federated login. Which means we’d have to implement a lot of different pages. I’d be willing to investigate it again, but I don’t have (and I don’t think any other Streamlink member has) a cable login to investigate with :)

For us to develop and maintain support for those cable logins, we’d need an account for each of them. But like I said, it’s not from lack of want but from lack of resources :)
